src/VsixV3/EditorsPackage/Microsoft.VisualStudio.Editors.swr

```diff
@@ -12,4 +12,7 @@ vs.localizedResources
                        description="Editors implementation for Project Properties"
 
 vs.payloads
-  vs.payload source=$(OutputPath)\VisualStudioEditorsSetup.vsix
\ No newline at end of file
+  vs.payload source=$(OutputPath)\VisualStudioEditorsSetup.vsix
+
+vs.dependencies
```

Comment:
Can you add a comment here that this is needed so that VSIXInstaller is around to uninstall it?